<?xml version="1.0"?>
<launch>

  <!-- <include file="$(find nav_pumas)/config_files/launch/navigation_real.launch"/> -->
  <node pkg="human_detector"  name="human_detector" type="human_detector_server.py"/>  
  <node pkg="face_recog"  name="face_recog" type="face_recog_server.py"/>
  <node pkg="face_recog"  name="face_recog_deep" type="face_recog_server_deep.py"/>
  <node pkg="object_classification"  name="yolo_classification" type="classification_server_receptionist_yoleado.py"/>
  <node pkg="segmentation"  name="segment_region" type="segment_region_server.py"/>
  <param name="map_yaml" value="$(find config_files)/prohibition_maps/map_lab_2024/map.yaml"/>
  <node pkg="face_recog"  name="analyze_face_backgnd" type="analyze_face_backgnd_node"/>
  <node pkg="head"  name="mic_node" type="mic_node"/>
  <node name="vosk_keywords" pkg="ros_whisper_vosk" type="vosk_node.py" respawn="true" output="screen" />
  <!-- <node pkg="smach_viewer"  name="smach_viewer" type="smach_viewer.py"/>   -->
  <!-- <node pkg="segmentation"  name="segmentator_node" type="segment_floor.py"/>   -->

<!--include file="$(find speech_recog)/launch/ros_vosk.launch"/> -->
  
<!-- </include> -->
<!-- <include file="$(find pocketsphinx)/launch/pocketsphinx.launch"> -->
  <!-- </include> -->
<!--node pkg="task" name="receptionist" type="Receptionist_gr.py"/-->
  
<!-- </include> -->
<!-- <include file="$(find pocketsphinx)/launch/pocketsphinx.launch"> -->
  <!-- </include> -->
<!--node pkg="task" name="receptionist" type="Receptionist_gr.py"/-->
</launch>

 
